src/store/modules/request/keys.js

```diff
@@ -197,6 +197,16 @@ const dataGetters = {
         result.push(audit);
     }
     return result;
+  },
+
+  // Returns a set containing just the form names that appear more than once
+  // in a project. Used on project overview to show form ID next to form name
+  // when form names are duplicated.
+  duplicateFormNames: ({ data: { forms, deletedForms } }) => {
+    const allForms = [...forms || [], ...deletedForms || []];
+    const formNames = allForms.map((form) => (form.nameOrId().toLowerCase()));
```

Comment:
How about using `toLocaleLowerCase()`? I like that we're using `localeCompare()` to sort form names in src/util/sort.js.